lib/os_io.c

```diff
@@ -37,6 +37,130 @@ ___io_module ___io_mod =
 
 /*---------------------------------------------------------------------------*/
 
+/* poll dynamic fdset memory management. */
+
+#ifdef USE_poll
+#define ___fdset_state_size(ps)     ps->os.fdset.size
```

Comment:
They are more of a historical artifact from how the implementation came to be; they were handy while the state changed from thread local vars to pstate members.
I can easily remove.